hack/verify.sh

```diff
@@ -64,6 +65,16 @@ function intergration_test() {
     else
         kind get kubeconfig --name=kind > /tmp/.kube/config
     fi
+    until ${kepler_ready} ; do
+        kubectl logs $(kubectl -n kepler get pods -o name) -n kepler > kepler.log
+	if [ `grep -c "Started Kepler in" kepler.log` -ne '0' ]; then
```

Comment:
can you also check if `Successfully load eBPF module` is there too? ebpf could fail to load yet Kepler can still start

Reply:
One question, if ebpf really fail, how could we deal with it here? echo errmsg and exit 1?
Per current code logic, will be endless loop.